praetorian_cli/sdk/entities/search.py

```
@@ -462,6 +464,76 @@ def fulltext(self, term, kind=None, limit=100) -> tuple:
+        :param anchor_key: exact key of the starting node
+        :param labels: list of relationship labels to follow (e.g.
+            ['HAS_WEBPAGE', 'HAS_VULNERABILITY']). Omit to follow every known edge
+            type -- the backend has no wildcard, so we OR the full label set.
+        :type labels: list
+        :param neighbor_kind: optional Kind to constrain the neighbor (e.g. 'risk')
+        :param optional: keep entries whose edge didn't match (rarely useful here)
+        :return: [{label, edge, target}], one per matched edge; 'edge' holds the
+            relationship's own properties, 'target' is the neighbor node.
+        :rtype: list
+        """
+        rel_labels = [Relationship.Label(l) for l in labels] if labels else list(Relationship.Label)
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)